database_schemas/cassandra/benchflow.cql

```diff
@@ -233,6 +234,32 @@ CREATE TABLE exp_cpu (
   PRIMARY KEY (experiment_id, container_id)
 );
 
+CREATE TABLE exp_cpu_core (
```

Comment:
@VincenzoFerme I looked into this again, and I think unless the lists become gigantic (which is unlikely) then yes, we are safe using lists instead of having one row per core.